test/unit/test_validate.py

```diff
@@ -24,3 +30,20 @@ def test_negentropy_uniform() -> None:
 def test_negentropy_zero() -> None:
     negentropy = validate.negentropy(np.zeros(100))
     assert negentropy == -np.inf
+
+
+def test_negentropy_maximizer_explicit() -> None:
+    maximizer = validate.ScalarMaximizer(objective=parabolic_objective)
```

Comment:
should we have a test that checks whether the golden method vs range method yield a similar result? Or do you think this is redundant with the two individual tests?

Reply:
feels redundant to me, since we confirm both are accurate by comparing to an analytical result